Add ESP32 support (#28)
cloud4rpi/docs: README.md

You can read it on [Cloud4RPi Documentation](http://docs.cloud4rpi.io) website.

> The documentation is created with [MkDocs](http://www.mkdocs.org/). If you need to open it locally do the following:

Reviewer: ```suggestion
The documentation is created with [MkDocs](http://www.mkdocs.org/). Follow the steps below to open it locally. 
```

> - Install [MkDocs](http://www.mkdocs.org/#installation) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) using the `pip install mkdocs mkdocs-material` command.

Reviewer: ```suggestion
- Use the `pip install mkdocs mkdocs-material` command to install [MkDocs](http://www.mkdocs.org/#installation) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
```

- Run the [serve.sh.bat](serve.sh.bat) file.

> While **MkDocs** is running, you should be able to access the documentation using the [http://127.0.0.1:8000](http://127.0.0.1:8000) URL.

Reviewer: ```suggestion
While **MkDocs** is running, use the [http://127.0.0.1:8000](http://127.0.0.1:8000) URL to access the documentation.
```
